wurst/_handles/Dialog.wurst

```diff
@@ -0,0 +1,43 @@
+package Dialog
+import NoWurst
+
+/**
+Dialogs are big dialog boxes at the center of the screen. The player can choose one button to click.
+Dialogs pause the game in single player mode.
+In multiplayer mode dialogs do not pause the game, but prevent players, who see the dialog from playing the game.
+**/
+
+public function createDialog() returns dialog
+    return DialogCreate()
+
+public function dialog.addButton(string buttonText) returns button
+    return this.addButton(buttonText, 0)
+
+/**Hotkey: use ASCII numbers of the capital letter.*/
+public function dialog.addButton(string buttonText, integer hotkey) returns button
+    return DialogAddButton(this, buttonText, hotkey)
+
+/**If clicked, quits the game*/
```

Comment:
Should be more like "adds a quit button to this dialog. If it is clicked, it ends the game for that player(?)"